selfdrive/car/docs_definitions.py

```diff
@@ -20,7 +21,7 @@ class Column(Enum):
   FSR_STEERING = "No ALC below"
   STEERING_TORQUE = "Steering Torque"
   AUTO_RESUME = "Resume from stop"
-  HARNESS = "Harness"
+  HARNESS = "<img width=2000>Harness Kit<img width=2000>"
```

Comment:
only need one img, <br> and space for under? Also try to make the link ="#" so it doesn't refresh

Reply:
I want to keep template specific stuff in the actual template. Can we do that?

Reply:
> I want to keep template specific stuff in the actual template. Can we do that?

Yes, but it is very ugly in jinja. Look at the new template